Game/Assets/Coliseo/Scripts/AdvVRTracker.cs

```diff
@@ -0,0 +1,165 @@
+﻿using System.Diagnostics;
+using System.IO;
+using UnityEngine;
+
+public class AdvVRTracker : MonoBehaviour {
+
+    UnityEngine.VR.VRNode VRHead = UnityEngine.VR.VRNode.Head;
+    TextWriterTraceListener listener;
+    Transform cameraTransform;
+    Transform cameraRigTransform;
+    Transform headTransform;
+    Transform spineTransform;
+    Transform neckTransform;
+    Animator anim;
+
+    Vector3 rotationNeeded;
+    Vector3 positionNeeded;
+
+    Vector3 initHeadRot;
+    
+    bool logging = false;
+
+    // These are all specific to the scientist model
```

Comment:
What scientific model? We should probably have a bit more description on how we picked the numbers here.